### PR TITLE
Kevin/ENG-928

### PR DESCRIPTION
**Pull request by kcvan**
## Description
Why did you write this code?
accommodate for rentivo bookings through trips receipt

to be used with: https://github.com/thebeetoken/beenest-backend/pull/709
## How to Test
- [ ] create a rentivo booking in thailand I(not mine)
- [ ] confirm that the trips receipt page looks good

Which devices did you test on?
- [ ] Chrome on Mac
- [ ] Chrome on PC
- [ ] Firefox on Mac
- [ ] Firefox on PC
- [ ] Safari iPhone
- [ ] Chrome Android

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
Will this have future work?

## Learnings
Did you learn anything here you want to share with the team?

### CODE REVIEW

**Comment by tc** [comment]
can you uncommit your package-lock if no packages are added/removed?

**Comment by jerminatorhits** [comment]
(out of scope of this PR) I noticed that these was a blank security deposit when booking:

image

 we should prob make provide a value of 0 or not render the security deposit at all if it doesn't exist

**Comment by kcvan** [comment]
fixed the security deposit

File: src/components/routes/Trips/TripsReceipt/TripsReceipt.tsx [hunk 1]
```diff
@@ -146,10 +146,11 @@ const TripsReceiptsMeta = (props: Booking): JSX.Element => {
                   <span>
                     {addressLine1 && `${addressLine1}, `}
```
Comment (tc) [inline src/components/routes/Trips/TripsReceipt/TripsReceipt.tsx]: don't we have a address formatter for this?

Comment (kcvan) [reply src/components/routes/Trips/TripsReceipt/TripsReceipt.tsx]: fixed! updated the formatter with the newer one from backend

**Pull request merged**